pyaedt/modules/PostProcessor.py

```diff
@@ -4911,3 +4916,454 @@ def sample_ami_waveform(
+            while os.path.exists(export_file):
+                file_name = generate_unique_name(base_name)
+                export_file = os.path.join(path, file_name + ".csv")
+        if setup_name is None:
+            setup_name = "{} : {}".format(self._app.get_setups()[0], self._app.solution_type)
+        if timestep is None:
+            timestep = ""
+            if self._app.solution_type == "Transient":
+                self._app.logger.warning("No timestep specified. First timestep will be exported.")
+        else:
+            if not self._app.solution_type == "Transient":
+                self._app.logger.warning("Simulation is steady-state, timestep argument is ignored.")
+                timestep = ""
+        if design_variation is None:
+            design_variation = ""
+        self._app.osolution.ExportFanOperatingPoint(
+            [
+                "SolutionName:=",
+                setup_name,
+                "DesignVariationKey:=",
+                design_variation,
+                "ExportFilePath:=",
+                export_file,
+                "Overwrite:=",
+                True,
+                "TimeStep:=",
+                timestep,
+            ]
+        )
+        with open(export_file, "r") as f:
+            reader = csv.reader(f)
+            for line in reader:
+                if "Fan Instances" in line:
+                    vol_flow = line[1]
+                    p_rise = line[2]
+                    break
+            var = {line[0]: [float(line[1]), float(line[2])] for line in reader}
+        return [export_file, vol_flow, p_rise, var]
+
+    @pyaedt_function_handler
+    def _parse_field_summary_content(self, fs, setup_name, design_variation, quantity_name):
+        content = fs.get_field_summary_data(sweep_name=setup_name, design_variation=design_variation)
+        pattern = r"\[([^]]*)\]"
+        match = re.search(pattern, content["Quantity"][0])
+        if match:
+            content["Unit"] = [match.group(1)]
+        else:  # pragma: no cover
+            content["Unit"] = [None]
+
+        if quantity_name in TOTAL_QUANTITIES:
+            return {i: content[i][0] for i in ["Total", "Unit"]}
+        return {i: content[i][0] for i in ["Min", "Max", "Mean", "Stdev", "Unit"]}
+
+    @pyaedt_function_handler()
+    def evaluate_faces_quantity(
+        self,
+        faces_list,
+        quantity_name,
+        side="Default",
+        setup_name=None,
+        design_variation={},
+        ref_temperature="",
+    ):
+        """Export the field surface output.
+
+        Parameters
+        ----------
+        faces_list : list
+            List of faces to apply.
+        quantity_name : str
+            Name of the quantity to export.
+        side : str, optional
+            Select which mesh faces side to use. Available options are ``"Default"``, ``"Adjacent"``,
+            and ``"Combined"``. Default is ``"Default"``.
```

Comment:
```suggestion
            Which side of the mesh face to use. The default is ``Default``.
            Options are ``"Adjacent"``, ``"Combined"``, and ``"Default"``.
```
I'm hoping my suggestion is correct. I can't tell what is meant by "mesh faces side" because it seems that to use this you would construction, which is awkward, you would have to use the possessive form: "mesh face's side"

Reply:
Your suggested change makes perfect sense :)